### ISSUE TITLE
Allow support for checking boxes from the main tasks screen 

### ISSUE THREAD

**Issue by ThinkDigitalSoftware** [SolarLune/masterplan#16]
Currently we have to open the task to check it.

**Comment by exelotl** [comment]
You can check/uncheck the currently selected task(s) by pressing "C".

But I must admit, my intuition makes me want to hit "space" to do this instead. Also I guess it wouldn't do any harm to make it so that clicking on the box-icon for the task on the main screen does it too?

**Comment by ThinkDigitalSoftware** [comment]
I agree with this. We're moving away from using letters as primary source of interaction as general users. I would go with the clicking of the box and or checkbox as an included way of checking these.
